### ISSUE TITLE
Advice on value of k

### ISSUE THREAD

**Issue by InleEfrafa** [Energy-Pathways-Group/GLOceanKit#91]
Do you have any guidance for the constant value of K to choose for the EVP whilst attempting to determine RMS values for vertical for a given stratification profile?

Many thanks

**Comment by JeffreyEarly** [comment]
Yes. If you are just looking to project a given density or velocity profile, you'll want to assume the hydrostatic modes. That is effectively like setting k=0.

**Comment by JeffreyEarly** [comment]
It depends on the particular dynamical variable you're interested in, and whether you're interested in waves or geostrophic motions. Check out [equations 6 and 9](https://agupubs.onlinelibrary.wiley.com/doi/epdf/10.1029/2019MS001939) for the correct usage.

Does that answer your question?
